src/main/java/org/takes/http/BkBasic.java

```diff
@@ -71,18 +71,29 @@ public BkBasic(final Take tks) {
         this.take = tks;
     }
 
+    @SuppressWarnings ("PMD.AvoidInstantiatingObjectsInLoops")
     @Override
     public void accept(final Socket socket) throws IOException {
         final InputStream input = socket.getInputStream();
+        final BufferedOutputStream output = new BufferedOutputStream(
```

Comment:
@sebing let's use try-with-resources feature here since we're now on Java 7. Make sure to include both `input` and `output`.